Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -1433,7 +1433,7 @@ void NetworkConnectionToWebProcess::logOnBehalfOfWebContent(IPC::DataReference&&
     }
     RELEASE_ASSERT(logType == OS_LOG_TYPE_DEFAULT || logType == OS_LOG_TYPE_INFO || logType == OS_LOG_TYPE_DEBUG || logType == OS_LOG_TYPE_ERROR || logType == OS_LOG_TYPE_FAULT);
 
-    os_log_with_type(osLogChannel.get() ? osLogChannel.get() : OS_LOG_DEFAULT, static_cast<os_log_type_t>(logType), "WebContent[pid=%d, identifier=%llu]: %s", pid, m_webProcessIdentifier.toUInt64(), logString.data());
+    os_log_with_type(osLogChannel.get() ? osLogChannel.get() : OS_LOG_DEFAULT, static_cast<os_log_type_t>(logType), "WebContent[pid=%d, identifier=%llu]: %{public}s", pid, m_webProcessIdentifier.toUInt64(), logString.data());
```

Comment:
This is a great place to explain why it's OK to use `{public}`, so that people don't have to go back to the commit message in the future.

Reply:
> This is a great place to explain why it's OK to use `{public}`, so that people don't have to go back to the commit message in the future.

Ah, good point, will add.

Thanks for reviewing!